pages/linux/mkfs.ext4.md

```diff
@@ -0,0 +1,11 @@
+# mkfs.ext4
+
+> Creates an ext4 filesystem inside a partition.
+
+- Create an ext4 filesystem inside partition 1 on device b (`sdb1`):
+
+`mkfs.ext4 {{/dev/sdb1}}`
```

Comment:
I'm pretty sure this command requires root privileges. On other pages like this, we've prefixed the examples with `sudo` to signify this. Perhaps we could do the same here?
```suggestion
`sudo mkfs.ext4 {{/dev/sdb1}}`
```